_data/projects/auth-jwt.yml

```diff
@@ -0,0 +1,13 @@
+---
+name: auth-jwt
+desc: 'A demo to learn JWT by reverse engineering'
```

Comment:
' is not needed here.
```suggestion
desc: A demo to learn JWT by reverse engineering
```

Reply:
Fixed in latest commit. Can you please review again.

Reply:
Ohk